### DISCUSSION TITLE
Better overview in custom domains tab for subbdomains

### DISCUSSION THREAD

**Discussion by Nalci** [simple-login/app#1828] [Feature Requests]
I'm using many subdomains for different occasions.

At the moment the overview in the custom domains tab is very confusing with many existing (sub)domains.

I wish the main domain would be at first and the associated subdomains would be below. 
If you click on the main domain or on a link like "show subdomains" the subdomains would show like a list below with all relevant informations.

If the main domain has not been added to simplelogin, maybe simplelogin could recognize automatically the main domain and would show also the list below.

The example image is kept very simple, I don't know any better :(

![Screenshot_20230801_101458_Skiff Pages](https://github.com/simple-login/app/assets/95542245/a2b0d03e-ae93-45e8-900b-c1ca7bb2c22a)

**Comment by nguyenkims** [comment]
Hi

By subdomain, do you mean a) subdomains for your own domain or b)the ones that SimpleLogin provides? If a), currently we treat the subdomain like a normal domain indeed and their settings are completely independent. Grouping them together might give users the feeling that they are linked but in reality they aren’t.

Best,
Son NK. SimpleLogin & Proton Pass

**Reply by Nalci** [reply]
Hey

By subdomain I meant a) my own domain. I didn't thought about that it may confuse users. It's just a bit messy right now if you have many domains.

But I understand :) Thanks for your reply!
